rpcs3/emucore.vcxproj.filters

```diff
@@ -1869,9 +1869,21 @@
     </ClInclude>
     <ClInclude Include="Emu\Cell\Common.h">
       <Filter>Emu\CPU\Cell</Filter>
-    </ClInclude>
+    </ClInclude>
```

Comment:
If you fix conflicts, it's better to discard all changes in conflicted `.filters` file, reload the solution and then move incorrectly placed files in solution explorer.